app/controllers/ProjectSidewalkAPIController.scala

```diff
@@ -63,20 +84,35 @@ class ProjectSidewalkAPIController @Inject()(implicit val env: Environment[User,
     * @param lat2
     * @param lng2
     * @param severity
+    * @param filetype
     * @return
     */
-  def getAccessAttributesWithLabelsV2(lat1: Double, lng1: Double, lat2: Double, lng2: Double, severity: Option[String]) = UserAwareAction.async { implicit request =>
+  def getAccessAttributesWithLabelsV2(lat1: Double, lng1: Double, lat2: Double, lng2: Double, severity: Option[String], filetype: Option[String]) = UserAwareAction.async { implicit request =>
     apiLogging(request.remoteAddress, request.identity, request.toString)
 
     val minLat:Float = min(lat1, lat2).toFloat
     val maxLat:Float = max(lat1, lat2).toFloat
     val minLng:Float = min(lng1, lng2).toFloat
     val maxLng:Float = max(lng1, lng2).toFloat
 
-    val features: List[JsObject] =
-      GlobalAttributeTable.getGlobalAttributesWithLabelsInBoundingBox(minLat, minLng, maxLat, maxLng, severity).map(_.toJSON)
-
-    Future.successful(Ok(Json.obj("type" -> "FeatureCollection", "features" -> features)))
+    if (filetype != None && filetype.get == "csv") {  // in CSV format.
+      val file = new java.io.File("access_attributes_with_labels.csv")
+      val writer = new java.io.PrintStream(file)
+      val header: String = "Neighborhood Name,Region ID,Access Score,Coordinates,Coverage,Average Curb Ramp Score," + 
+                            "Average No Curb Ramp Score,Average Obstacle Score,Average Surface Problem Score," + 
+                            "Curb Ramp Significance,No Curb Ramp Significance,Obstacle Significance," + 
+                            "Surface Problem Significance"
+      writer.println(header)  // Write column headers.
```

Comment:
In general, we prefer having comments on their own line. It can make sense sometimes (like if you want to have a comment before an else clause, it can look weird to have it the line before). But in this case (and most of the time) can you put the comments on their own line?

Reply:
Another tiny note: I'd include this comment on the line before you create the `header` val.

Reply:
I updated the comment style.